.github/workflows/release.yml

```diff
@@ -45,19 +45,63 @@ jobs:
         name: pganalyze-collector-linux-amd64
         path: pganalyze-collector-linux-amd64
 
+  build_packages:
+    # Use older Ubuntu release to ensure availability of CGroupsv1 (which are necessary
+    # currently to support older systemd on some of the test containers)
+    runs-on: ubuntu-20.04
+    permissions: {}
+    # ensure we don't release on create events for branches (only tags)
+    if: ${{ startsWith( github.ref, 'refs/tags' ) }}
+
+    steps:
+
+    - name: Check out code
+      uses: actions/checkout@v3
+      with:
+        submodules: true
+
+    - name: Set up QEMU
+      uses: docker/setup-qemu-action@v1
+
+    - name: Build packages
+      run: |
+        DOCKER_BUILDKIT=1 make -C packages build
+
+    - name: Test packages
+      run: |
+        DOCKER_BUILDKIT=1 make -C packages test
```

Comment:
Doing the package test here (instead of only the build) adds ~2hrs to the release task - which is not great.

However I'm not sure the alternative is a good idea (releasing without the test), so I'd suggest we do this for now, and if we get lots of issues with this we revise.

Previously I used to run this locally during the package release, so this matches the existing release process.